images/Dockerfile

```diff
@@ -0,0 +1,21 @@
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+ARG RELEASE_VER=4.6
+ARG INSTALL_DIR=/root/ocp-install
+
+ENV PATH $PATH:$INSTALL_DIR
+
+WORKDIR $INSTALL_DIR
+
+RUN microdnf update && \
+    microdnf install -y yum findutils && \
+    mkdir -p $INSTALL_DIR
+
+RUN yum install -y openssh openssh-clients
+
+RUN curl -sL https://raw.githubusercontent.com/ocp-power-automation/openshift-install-power/master/openshift-install-powervs -o /usr/bin/openshift-install-powervs && \
```

Comment:
Why not COPY/ADD from current cloned directory?

Reply:
Modified the Dockerfile